Update cheatsheet links for intro-R and scRNAseq (#521)
AlexsLemonade/training-modules: module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

##### Each table represents a different library/tool and the corresponding commands. 
> Please note that these tables are not intended to tell you all the information you need to know about each command. 
> 

> > The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command. Please be aware that the documentation will generally provide information about the given function's most current version, which will usually (but not always!) match what you have installed on your machine. If you have certain older versions or R or other R packages, the documentation may differ from what you have installed.

Reviewer: To ease future revisions, we try to follow a one sentence per line convention (you may see this abbreviated as OSPL). I have changed that here, as well as making a small change since I know that rdrr does not always have the latest version. 
```suggestion
> The hyperlinks found in each piece of code will take you to the documentation for further information on the usage of each command.
Please be aware that the documentation will generally provide information about the given function's most current version (or a recent version, depending on how often the documentation site is updated).
This will usually (but not always!) match what you have installed on your machine.
If you have a different version of R or other R packages, the documentation may differ from what you have installed.
```

|------------------------------------|----------------------|---------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
| Base `R`          | [`library()`](https://rdrr.io/r/base/library.html)                    | Library   | Loads and attaches additional packages to the R environment.                                                                                                                                                  |
| Base `R`          | [`<-`](https://rdrr.io/r/base/assignOps.html)                    | Assignment operator   | Assigns a name to something in the R environment.                                                                                                                                                  |
| Base `R`          | [`c()`](https://rdrr.io/r/base/c.html)                     | Combine           | Combines values into a vector or list.                                                                                                                                                             |

> | Base `R`          | [`%in%`](http://www.datasciencemadesimple.com/in-operator-in-r/)                                  | "in" logical operator           | Checks if the given value(s) on the left side of the operator are in the vector or other R object defined on the right side of the operator. It returns a logical `TRUE` or `FALSE` statement.                                                                                                                                                              |

Reviewer: ```suggestion
| Base `R`          | [`%in%`](http://www.datasciencemadesimple.com/in-operator-in-r/)                                  | "in" logical operator           | Checks if the given value(s) on the left side of the operator are in the vector or other R object defined on the right side of the operator. It returns a logical `TRUE` or `FALSE` statement.                                                                                                                                                              |
```

Do we want to link to https://rdrr.io/r/base/match.html for uniformity? We could link to both...

Author: I actually went back and forth on this one. I liked the `datasciencemadesimple` more because it was just about the operator, but point taken about consistency. I think both is a good call for this one. I can change the link to rdrr, and add into description `[This](datsciencemadesimple) is also useful` or similar.

| `dplyr`           | [`group_by()`](https://dplyr.tidyverse.org/reference/group_by.html)        | Group By              | Groups data into rows that contain the same specified value(s)                                                                                                                                    |
| `dplyr`           | [`inner_join()`](https://dplyr.tidyverse.org/reference/mutate-joins.html)          | Inner Join            | Joins data from two data frames, retaining only the rows that are in both datasets.    |

<div style="page-break-after: always;"></div>
                                                                                                                                                                                                                 
#### `ggplot2`
Read the `ggplot2` package documentation [**here**](https://ggplot2.tidyverse.org/). <br>

> A vignette on the usage of the `ggplot2` aesthetics can be found [**here**](https://ggplot2.tidyverse.org/articles/ggplot2-specs.html). Additional vignettes are available from the "Articles" dropdown menu on this webpage.

Reviewer: OSPL
```suggestion
A vignette on the usage of the `ggplot2` aesthetics can be found [**here**](https://ggplot2.tidyverse.org/articles/ggplot2-specs.html). 
Additional vignettes are available from the "Articles" dropdown menu on this webpage.
```
